### PR TITLE
SATT-60: React component 1.4.1 which fix listing prices

### PR DESCRIPTION
**Pull request by tvalimaa**
### Description

React component update 1.4.1 which fix listing pages empty price problem 
https://andersinno.atlassian.net/browse/SATT-60

### How to test it

- make fresh or make build drush-cr
- rebuild tracking information search api apartment listing https://asuntotuotanto.docker.so/fi/admin/config/search/search-api/index/apartment_listing
- go Haso listing page https://asuntotuotanto.docker.so/fi/asuntohaku/haso
- if you see apartment with empty Asumisoikeusmaksu
  - open a new tab apartment by click tutustu
  - click edit and then click Hintatiedot tab
  - check that Luovutushinta and Alkuperäinen luovutushinta is empty
    - if both values are empty then everything is ok
  - Fill both values and save node (apartment)
- on terminal
  - make shell and then drush cron
- now refresh Haso listing page https://asuntotuotanto.docker.so/fi/admin/config/search/search-api/index/apartment_listing
  - Now listing page should show Luovutushinta value on Asumisoikeusmaksu column
- Find another apartment which shows empty on Asumisoikeusmaksu 
  - Click again Tutustu and edit that apartment node
  - Now fill only Alkuperäinen asumisoikeusmaksu and leave Luovustuhinta field empty
  - Save node
- Now again on terminal make shell and then drush cron
- refresh again haso listing page and now that Alkuperäinen asumisoikeusmaksu value should be on Asumisoikeusmaksu column

In short:
- Haso listing page should show Luovutushinta value on Asumisoikeusmaksu column unless Luovutushinta field is empty so then Alkuperäinen asumisoikeusmaksu will showing on Asumisoikeusmaksu column

### CODE REVIEW

**Review by OscarJonasson — APPROVED** [review]
Works as described, Approved
